sdk/core/core-auth/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.3.0 (Unreleased)
 
 - Adds the `AzureNamedKeyCredential` class which supports credential rotation and a corresponding `NamedKeyCredential` interface to support the use of static string-based names and keys in Azure clients.
+- Adds the `isNamedKeyCredential` and `isSASCredential` typeguard functions.
```

Comment:
Nit: 

```suggestion
- Adds the `isNamedKeyCredential` and `isSASCredential` typeguard functions similar to the existing `isTokenCredential`.
```

Reply:
Feel free to ignore this suggestion or add it as part of #14423 if you don't want to wait until another round of CI to merge this PR

Reply:
Will add it #14423